internal/pkg/cli/deploy/workload.go

```diff
@@ -544,6 +599,8 @@ type UploadArtifactsOutput struct {
 	AddonsURL                      string
 	CustomResourceURLs             map[string]string
 	StaticSiteAssetMappingLocation string
+	ImageNames                     []string
+	ContainerNames                 []string
```

Comment:
would it be easier to just add `imageName` to `ContainerImageIdentifier`? (in the `ImageDigests` map)